paper-input-container.html

```diff
@@ -365,10 +366,14 @@
       var value = inputElement[this._propertyForValue] || inputElement.value;
 
       var valid;
-      if (inputElement.validate) {
-        valid = inputElement.validate(value);
-      } else if (inputElement.checkValidity) {
-        valid = inputElement.checkValidity();
+
+      // type="number" doesn't have a validate method...
+      if (this.autoValidate) {
+        if (inputElement.validate) {
+          valid = inputElement.validate(value);
+        } else {
+          valid = inputElement.checkValidity();
+        }
       } else {
         valid = true;
```

Comment:
Not super sure, because `type="number"` is broken right now (because of https://github.com/PolymerElements/iron-input/issues/12), so typing anything breaks the element.